cpu/stm32/include/candev_stm32.h

```diff
@@ -42,7 +42,7 @@ extern "C" {
 #elif defined(CPU_FAM_STM32F1) || defined(CPU_FAM_STM32F2) || defined(CPU_FAM_STM32F4)
 #define CANDEV_STM32_CHAN_NUMOF 2
 #elif defined(CPU_FAM_STM32F0) || defined(CPU_FAM_STM32F3) || \
-      defined(CPU_FAM_STM32L4) || DOXYGEN
+      defined(CPU_FAM_STM32L4) || defined(CPU_LINE_STM32F722xx) || DOXYGEN
```

Comment:
Do you know if its only for this `stm32f7` or if its for all of them?

Reply:
> Do you know if its only for this `stm32f7` or if its for all of them?

STM32f7 MCU series distributes # of CAN interfaces specifically.
STM32F722/32/23/33: 1 x CAN
STM32F745/46/56: 2 x CAN
STM32F765/67/69/77/79: 3 x CAN

reference: https://www.st.com/en/microcontrollers-microprocessors/stm32f7-series.html

For Nucleo-f722ze board, it falls into CPU_LINE_STM32F722xx

After this done, i will add CAN for nucleo-f767zi ([#15710](https://github.com/RIOT-OS/RIOT/issues/15710)) that has 3 CANs.

Reply:
Thanks perfect.